packages/constants/src/mappings/districts.ts

```diff
@@ -0,0 +1,58 @@
+import {DISTRICTS as MCF_DISTRICTS} from '../mcf';
+import {DISTRICTS as ICMS_DISTRICTS} from '../icms';
+import {DISTRICTS as MSF_DISTRICTS} from '../msf';
+
+// ICMS <-> MCF transformation
+const ICMS_MCF_MAPPING_TABLE: Array<[number, number]> = [
+  [1, 1],
+  [2, 2],
+  [3, 3],
+  [4, 4],
+  [5, 5],
+  [6, 6],
+  [7, 7],
+  [8, 8],
+  [9, 9],
+  [10, 10],
+  [11, 11],
+  [12, 12],
+  [13, 13],
+  [14, 14],
+  [15, 15],
+  [16, 16],
+  [17, 17],
+  [18, 18],
+  [19, 19],
+  [20, 20],
+  [21, 21],
+  [22, 22],
+  [23, 23],
+  [24, 24],
+  [25, 25],
+  [26, 26],
+  [27, 27],
+  [27, 27],
+  [28, 28],
+  [28, 28],
+  [9999, 998],
```

Comment:
there are 2 duplications.

Implementation wise it may be easier to not use a mapping table and just handle the specific use case of 9999 <=> 998

Reply:
sounds gd. I have removed the mapping table